tools/health/isAlive

```diff
@@ -85,13 +85,6 @@ def isElkAlive() :
     okString = '"cluster_name"'
     return monitorUtil.outputChecker(cmd, okString, 1)
 
-def isKafkaAlive() :
```

Comment:
if we remove this we have no way to check if kafka is alive and responding. 
can we replace this with a producer and consumer pair on a dedicate health topic?